indexd/drs/blueprint.py

```diff
@@ -180,11 +184,12 @@ def indexd_to_drs(record, expand=False):
 
     drs_object = {
         "id": did,
-        "description": "",
         "mime_type": "application/json",
         "name": name,
+        "index_created_time": index_created_time,
+        "index_updated_time": index_updated_time,
         "created_time": created_time,
-        "updated_time": updated_date,
+        "updated_time": updated_time,
```

Comment:
Thanks for the tip. I've updated the new features description. While rewording it, I realized this might actually be considered a breaking change so I updated that section as well. LMK if I should pull that out.